### ISSUE TITLE
[Feature Request] option to make images is next to text inside threads 

### ISSUE THREAD

**Issue by baraa272** [K1rakishou/Kuroba-Experimental#354]
as name of request said, in original clover and old kuroba the default is the image is next to text inside threads while in kurobaEX its image above text inside threads 
so can you please add an toggle for it ??
with all love and respect

**Comment by K1rakishou** [comment]
As I have already answered in the thread I don't like the previous version because it leaves the empty space on the left side, under the thumbnail, and on phones horizontal space is more valuable than vertical. I gave it some thought and here are two variants I came up with but they still suck.

![image](https://user-images.githubusercontent.com/10332458/115855737-46eb1c00-a434-11eb-84b8-fa212cadabd9.png)

The first variant takes waaay too much horizontal space and I don't like the second variant because file names may be really long so it will get wrapped into a multiline text and will make a thumbnail take even more vertical space. Also there are sites that support up to 8 images per post so you can guess how bad it will look.

In other words this thing simply won't work in this app anymore unless I revert everything back which I won't do, obviously. At least I couldn't come up with a good solution. If you don't like the thumbnail taking space you can make it smaller in the settings (and even disable file names).

**Comment by baraa272** [comment]
commenting on the wasted space i really really don't think it matters ON high DPI devices 
i have an S7 edge with 500dpi and really really think the space doesn't count at all, i can view anything comfortably 
i really hope you add the option since old kuroba is out of support on android 11 and I'm moving to a new phone 
i really really think its up to user to switch between modes as he wants 
also for notice mode 2 is better in your picture
also how to remove that 1px margin on the left on kurobaEX ???
i know its used for marking new replies but i think its taking so much there 
you should switch to the Blue line marking just like regular kuroba 
also the toolbar is taking so much horizontal space catalog 
you should move it somewhere else or make a special menu for it

**Comment by K1rakishou** [comment]
>i really really think its up to user to switch between modes as he wants

There are no "modes". This thing was first introduced in Kuroba and was put under a setting. Then I removed it completely during the early stage of KurobaEx development because I didn't like how it was initially implemented. Then when reworking post cell I decided to put text under the thumbnails because thumbnails got way mode wide than they were before because now the short image is displayed to the right of thumbnails. Now people are asking me to revert everything back because it doesn't look like Clover/Kuroba.

>also for notice mode 2 is better in your picture

Mode 2 won't work for multiple images. I understand that 4chan only has 1 picture per post but what do I do with other sites? Leaving them all in one column will make posts long as fuck.

>also how to remove that 1px margin on the left on kurobaEX ???

Disable "mark unseen posts"

>also the toolbar is taking so much horizontal space catalog

Toolbar is a standard UI component on Android so it's not going anywhere.

**Comment by baraa272** [comment]
> ow people are asking me to revert everything back because it doesn't look like Clover/Kuroba.

with all due respect to all efforts and work you put on the newest updated 4chan app, i dont see a problem here 
people have been using clover since forever and kuroba is just an updated clover fork so thats that 


> Mode 2 won't work for multiple images. I understand that 4chan only has 1 picture per post but what do I do with other sites? Leaving them all in one column will make posts long as fuck.

i really dont know here 



> Disable "mark unseen posts"

the blank space is there even after disabling it 

![Screenshot_2021-04-24-12-45-19](https://user-images.githubusercontent.com/32843097/115954811-7ce6d980-a4e2-11eb-8221-3928b7bd1e25.png)




> Toolbar is a standard UI component on Android so it's not going anywhere.

ok

**Comment by K1rakishou** [comment]
>people have been using clover since forever and kuroba is just an updated clover fork so thats that

Yeah and they assume that if this application has "Kuroba" in the name then it must look exactly the same as the other ones. Which I don't agree with because the whole point of this application is to try something new/different, hence the "experimental". I mean, I get that it may be hard for some people to get used to the changes, but if it's really that hard then don't use an app that is in heavy development mode, wait until it's more stabilized (which will definitely happen at some point in the future).

>the blank space is there even after disabling it

That space is also used by "highlight" filters. I will probably rework it at some point.

>ok

I mean they (toolbar/bottom nav view) both disappear when you scroll down so it shouldn't be a problem most of the time since you are supposed to use the app by constantly scrolling down. And they literally take zero space once they are hidden.

**Comment by baraa272** [comment]
>  I mean, I get that it may be hard for some people to get used to the changes, but if it's really that hard then don't use an app that is in heavy development mode, wait until it's more stabilized (which will definitely happen at some point in the future).

really ?, i thought the app will stay like that lmao my apologies 



> That space is also used by "highlight" filters. I will probably rework it at some point.

yeah it needs a rework 



> I mean they (toolbar/bottom nav view) both disappear when you scroll down so it shouldn't be a problem most of the time since you are supposed to use the app by constantly scrolling down. And they literally take zero space once they are hidden.

guess i'll try to get used to it in my older phone 
also for note i have 2 phones 
S7 Edge Exynous and S5 Snapdragon
for my S7 i'm using an old kuroba dev version which kind of stable, (i really didnt like the current changes to kurobadev in latest versions) 
on my S5 i'm using your kurobaEx stable version (hence that scrrenshoot from) 
i'll be soon be buying an newer phone with android 11 and regarding that i fear that i wont be able to use the current older version of kurobadev due to incompatibility issues 
thats why i'm hoping kurobaEx will be better since its being updates for the newer versions of android

**Comment by K1rakishou** [comment]
>i'll be soon be buying an newer phone with android 11 and regarding that i fear that i wont be able to use the current older version of kurobadev due to incompatibility issues

I think there shouldn't be any problems. It supports SAF, which is the main pain the ass in Android 11, and everything else should work too. And even if it doesn't IIRC it still targets Android 10 so it should be fine.

**Comment by baraa272** [comment]
thx for notice 
good luck dev hope everything works out well in the end
